middleware/sunset.go

```diff
@@ -0,0 +1,29 @@
+package middleware
+
+import (
+	"net/http"
+	"time"
+)
+
+// Sunset set Deprecation/Sunset header to response
+// This can be used to enable Sunset in a route or a route group
+// For more at: https://datatracker.ietf.org/doc/html/draft-ietf-httpapi-deprecation-header-02
```

Comment:
Looks like an Expired Internet-Draft. Is there any timeline if this makes it into an RFC?

Reply:
Looks like its in the RFC, can you please confirm this? Here is a link: https://www.rfc-editor.org/rfc/rfc8594.html#section-1.4

Also updated the link in the comment as well.